docs/langref/src/structs.md

```diff
@@ -1,11 +1,30 @@
 ## Structs
 
-Anonymous structs type can be declared with curly braces: `{ identifier1: type2, identifier1: type2, }`
-The trailing semicolon is optional.
-They can be initialized with a struct literal: `{ identifier1: expression1, identifier2: expression2  }`
+One can define named structures using the `struct` keyword,
+
+```slint,no-preview
+export struct Player  {
+    name: string,
+    score: int,
+}
+
+export component Example {
+    in-out property<Player> player: { name: "Foo", score: 100 };
+}
+```
+
+### Anonymous Structures
+
+`{ identifier1: type2, identifier1: type2 }` declares an anonymous structure.
+
+Struct literals like
+`{ identifier1: expression1, identifier2: expression2  }` can initialize anonymous
+structures. You may use a trailing `,` after `expression2` or in the anonymous structure
+definition above after `type2`.
 
 ```slint,no-preview
 export component Example {
+    // The trailing `;` is optional here:
```

Comment:
This is wrong!
Please remove this comment.

It is the trailing `,`  on the next line that is optional.

`{ identifier: `  always start a struct literal, not a scope.

Here we don't need the semi colon because of the extra braces:

```
    property<{name: string, score: int}> player: { { name: "Foo", score: 100 } }
```

Reply:
There is no optional `;`.  Only optional `,`.  
The previous docs were wrong (semicolon vs colon)